fuzzers/kirenenko/fuzzer.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for Kirenenko fuzzer"""
+import shutil
+import os
+import glob
+import subprocess
+import time
+import queue as Q
+from fuzzers import utils
+
+
+def is_benchmark(name):
+    "check if benchmark matches"
+    benchmark = os.getenv('BENCHMARK', None)
+    return benchmark is not None and name in benchmark
+
+
+def build_afl():
+    "build afl binary"
+    build_directory = os.environ['OUT']
+    os.environ['CC'] = '/afl/afl-clang-fast'
+    os.environ['CXX'] = '/afl/afl-clang-fast++'
+    os.environ['AFL_LLVM_USE_TRACE_PC'] = '1'
+    os.environ['AFL_LLVM_DICT2FILE'] = build_directory + '/afl++.dict'
+    env = os.environ.copy()
+    build_script = os.path.join(os.environ['SRC'], 'build.sh')
+    print("build script path is " + build_script)
```

Comment:
nit: Other than for docstrings (super confusing and arbitrary) we use single quotes for strings, so please change to 
`print('build script path is ' + build_script)`